tests/queries/0_stateless/03234_get_setting_or_default.sql

```diff
@@ -0,0 +1,26 @@
+SET custom_a = 'value_a';
+SET custom_b = 'value_b';
+SET custom_c = null;
+SET custom_d = 5;
+
+
+
+SELECT getSettingOrDefault('custom_a','default_a');
+SELECT getSettingOrDefault('custom_b','default_b');
+SELECT getSettingOrDefault('custom_c','default_c');
+SELECT getSettingOrDefault('custom_d','default_d');
+
+SELECT getSetting('custom_e');  -- { serverError UNKNOWN_SETTING }
```

Comment:
The case is already covered in another test, no need to have a duplicate here

Reply:
Right, Since I have refactored code specifically in this function's implementation, a test case is good for completeness.

Reply:
Yes, but we run all the tests anyway, to verify that nothing is broken. It doesn't make sense to have duplicates. However it doesn't really matter in this case, so I'm ok to leave it